8Knot/queries/query_template.py

```diff
@@ -1,22 +1,20 @@
 import logging
 import pandas as pd
-from db_manager.augur_manager import AugurManager
-from app import celery_app
-from cache_manager.cache_manager import CacheManager as cm
+import cache_manager.cache_facade as cf
 import io
-import datetime as dt
-from sqlalchemy.exc import SQLAlchemyError
 
 """
 TODO:
 (1) update QUERY_NAME
 (2) update 'NAME_query' found in function definition and in the function call that sets the 'ack' variable below.
```

Comment:
ack is no longer used update this to current structure

Reply:
Ah gotcha, will change